python/cpu_profile/job.slurm

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --job-name=matinv        # create a short name for your job
+#SBATCH --nodes=1                # node count
+#SBATCH --ntasks=1               # total number of tasks across all nodes
+#SBATCH --cpus-per-task=1        # cpu-cores per task (>1 if multi-threaded tasks)
+#SBATCH --mem-per-cpu=4G         # memory per cpu-core (4G is default)
+#SBATCH --time=00:01:00          # total run time limit (HH:MM:SS)
+#SBATCH --mail-type=begin        # send email when job begins
+#SBATCH --mail-type=end          # send email when job ends
+#SBATCH --mail-type=fail         # send email if job fails
+#SBATCH --mail-user=<YourNetID>@princeton.edu
+
+module purge
+module load anaconda3/2023.9
+conda activate 2024_02_workshop  # replace this with your environment containing line_profiler, or remove if in base
```

Comment:
Remove this - they can just install in base or add this eventually. The other examples don't specify envs.